src/calculation/ClauseResultsBuilder.ts

```diff
@@ -363,100 +372,215 @@ export function prettyResult(result: any | null, indentLevel?: number, keyIndent
   if (result instanceof DateTime) {
     return moment.utc(result.toString()).format('MM/DD/YYYY h:mm A');
   } else if (result instanceof Interval) {
-    return `INTERVAL: ${prettyResult(result.low)} - ${prettyResult(result.high)}`;
+    const typeStr = includeType ? 'INTERVAL: ' : '';
```

Comment:
Unrelated to this line, but for `DateTime`s we are using `MM/DD/YYYY h:mm A` and chopping off the seconds. Maybe we should just make it `MM/DD/YYYY h:mm:ss A` instead to show all the time values